added.. a basic style  div's etc

A-Lord opened the pull request:
and made tailwind components., also changed the side bar so it now has break points instead of a button.

A-Lord committed:
added.. a basic style

and made tailwind components., also changed the side bar so it now has break points instead of a button.

A-Lord committed:
format hotfix

Formated it

AhmadALshalabi committed:
add animation to pages when rendering

AhmadALshalabi reviewed (commented):
visit:  https://animate.style/
-------------------------------
steps to put animation on your page: 
1- npm install animate.css --save
2- import "animate.css"; //to your component
3- use "transition" buildin animation element 
4- use the class="enter-active-class="" // for "transition" 
5- add the desired animation style from the webpage 
6- enjoy!

test current animation > npm i

AhmadALshalabi commented on src/components/UI/ThemeToggle.vue:
```diff
@@ -1,30 +1,27 @@
+	<div @click="ToggleTheme" class="toggle">
+		<font-awesome-icon v-if="theme === 'light-theme'" icon="moon" class="text-xl text-[#fff] hover:text-[#2b2b2b]" />
+		<font-awesome-icon v-else icon="sun" class="text-xl text-[#ccf807]" />
+		<span class="sideBarText"> Theme </span>
```
tested without "Theme" word  and looks better

A-Lord commented:
Vi har ju redan animationer https://tailwindcss.com/docs/transition-property  =)

AhmadALshalabi commented:
> Vi har ju redan animationer https://tailwindcss.com/docs/transition-property =)

way eaiser to use the commented one :)

A-Lord commented:
> > Vi har ju redan animationer https://tailwindcss.com/docs/transition-property =)
> 
> way eaiser to use the commented one :)

kanske ., använd https://jofftiquez.github.io/v-animate-css/ .. för vue verisionen,.., men less is more när det gäller animationer för web.  Kollade och var inga av dom nya animationerna som passa på en budget sida. för flashiga :P

A-Lord committed:
hotfix screen size

AhmadALshalabi commented on src/assets/tailwind.css:
```diff
@@ -4,51 +4,108 @@
+		@apply flex flex-col justify-center;
+	}
+	textarea {
+		@apply mb-3 px-2 py-1.5 mb-3 mt-1 block w-full px-2 py-1.5 border border-gray-300 rounded-md text-sm shadow-sm placeholder-gray-400
```
double px, py (row 12) 
double px, py, mb (row 27 & 36)
dont know if this will cause problem with tailwind yet!

AhmadALshalabi replied:
fixed no worry

AhmadALshalabi committed:
delete double styling

A-Lord committed:
removed animation on router-views

removed transition on router-view, the animation times no mather how slow makes the site feel cluncky to use and laggy.  but added longer tranisition when u swap to darkmode/lightmode to get the feeling of turning off/on a light switch.
